doc.go

```diff
@@ -24,7 +24,13 @@ Use the Open() function to create a database handle with connection parameters:
 
 	db, err := sql.Open("snowflake", "<connection string>")
 
-The Go Snowflake Driver supports the following connection syntaxes (or data source name (DSN) formats):
+The Go Snowflake Driver supports the following connection syntaxes (or data source name (DSN) formats).
+The first three lines use the newer format (supported by the Organizations feature of Snowflake).
```

Comment:
Rather than putting the burden on the reader to count the number of lines, you should split this up into two separate lists.

```
If your account is under an organization
(link to an explanation of how you find this out),
use the following formats for the connection string:

- ...
- ...
- ...

Otherwise, use the following formats for the connection string:

- ...
- ...
- ...

```